tools/mo/openvino/__init__.py

```diff
@@ -7,7 +7,7 @@
 # Required for Windows OS platforms
 # Note: always top-level
 try:
-    from openvino.utils import _add_openvino_libs_to_search_path
+    from openvino.package_utils import _add_openvino_libs_to_search_path
```

Comment:
can we not touch it please because we are removing it this release?

Reply:
Unfortunately there are still some samples that depend on it. 
I had to touch it because `wheels_windows_vs2019_release` CI failed.